src/main/scala/scalaz/stream/Process.scala

```diff
@@ -737,8 +753,8 @@ object Process extends ProcessInstances {
   /** The `Process` which emits no values and signals normal termination. */
   val halt: Process0[Nothing] = halt0
 
-  /** Alias for `halt`. */
-  def empty[F[_],O]: Process[F, O] = halt
+  /** Empty emit. Used to signal end of evaluation form `step` to give chance step to be interrupted */
+  val empty: Emit[Nothing] = Emit(Nil)
```

Comment:
`empty` may be useful to help Scala with type inference. It is also nicer than writing `Process(): Process[Task, Int]` for example. See also #211